proposals/0031-hlsl-vector-matrix-operations.md

```diff
@@ -5,8 +5,8 @@
 ## Instructions
 
 - Proposal: [0031](0031-hlsl-vector-matrix-operations.md)
-- Author(s): [Damyan Pepper][damyanp], [Chris Bieneman][llvm-beanz], 
-             [Anupama Chandrasekhar][anupamachandra]
+- Author(s): [Damyan Pepper][damyanp], [Chris Bieneman][llvm-beanz],
```

Comment:
Apologies - I hit autoformat and it autoformatted it.  I could pull this out into another PR, but it seems a bit unnecessary since this is rapidly becoming a single big update anyway.

The text here hasn't changed in any meaningful way until line 114 (the first code example where the matrix layout enums have had their names changed).